jitbuilder/release/Makefile

```diff
@@ -43,6 +43,7 @@ test: goal
 	./controlflowtests
 	./issupportedtype
 	./toiltype
+	./atom
```

Comment:
maybe minor point: why is the executable called "atom" if the source code for the test has a different name?